### PR TITLE
Fix mouse drag position with winbar

### PR DESCRIPTION
**Pull request by zeertzjq**

### CODE REVIEW

**Comment by zeertzjq** [comment]
Still off-by-one with multigrid

**Comment by zeertzjq** [comment]
Hmm, without this PR clicks are off-by-one in multigrid, but with this PR drags are off-by-one in multigrid.

**Comment by zeertzjq** [comment]
Hmm, now the statsline dragging problem is back.

**Comment by zeertzjq** [comment]
I seem to have fixed mouse dragging position problem in floating window with border as well.

**Comment by zeertzjq** [comment]
Now this just needs test.

File: src/nvim/mouse.c [hunk 1]
```diff
@@ -159,16 +159,16 @@ int jump_to_mouse(int flags, bool *inclusive, int which_button)
...
       return IN_OTHER_WIN | (on_winbar ? MOUSE_WINBAR : 0);
     }
     on_winbar = false;
 
     // winpos and height may change in win_enter()!
-    if (grid == DEFAULT_GRID_HANDLE && row >= wp->w_height) {
+    if (grid == DEFAULT_GRID_HANDLE && row >= wp->w_height - wp->w_winbar_height) {
       // In (or below) status line
-      on_status_line = row - wp->w_height + 1;
+      on_status_line = row - wp->w_height + wp->w_winbar_height + 1;
```
Comment (zeertzjq) [inline src/nvim/mouse.c:171]: @bfredl Should `w_height_inner` be used here instead?

**Comment by zeertzjq** [comment]
This seems to fix mouse drag position in floating window with border in multigrid UI too.

**Review by bfredl — APPROVED** [review]

**Pull request merged**